### PR TITLE
Update install scripts to include ChangesEnvironment flag

### PR DESCRIPTION
**Pull request by garrytrinder**

### CODE REVIEW

File: scripts/local-setup.ps1 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-$versionString = "v0.18.0-beta.1"
+$versionString = "v0.18.0-beta.env"
```
Comment (waldekmastykarz) [inline scripts/local-setup.ps1:1]: Is this change intentional?

**Review by waldekmastykarz — CHANGES_REQUESTED** [review]

File: scripts/local-setup.ps1 (hunk 1 above)
Comment (garrytrinder) [reply scripts/local-setup.ps1:1]: No, good catch.

**Review by waldekmastykarz — APPROVED** [review]
